examples/cabtool.rs

```diff
@@ -42,21 +44,26 @@ fn main() {
     let cli = Cli::parse();
     match cli.command {
         Command::Cat { path, files } => {
-            let mut cabinet = Cabinet::new(File::open(path).unwrap()).unwrap();
+            let cabinet = Cabinet::new(File::open(path).unwrap()).unwrap();
             for filename in files {
-                let mut file_reader = cabinet.read_file(&filename).unwrap();
-                io::copy(&mut file_reader, &mut io::stdout()).unwrap();
+                while let Some(folder) = cabinet.folder_entries().next() {
```

Comment:
Is this line correct?  It seems like each loop iteration is creating a new `folder_entries()` iterator and starting over from the beginning.

(There's a few other loops that look like this too.)

Reply:
Ups, my bad. Good catch, I will fix it.